src/loaders/OmeZarrLoader.ts

```diff
@@ -234,16 +234,21 @@ function convertChannel(channelData: TypedArray, dtype: string): Uint8Array {
   // get min and max
   let min = channelData[0];
   let max = channelData[0];
-  channelData.forEach((val: number) => {
-    min = Math.min(min, val);
-    max = Math.max(max, val);
-  });
+  for (let i = 0; i < channelData.length; i++) {
+    const val = channelData[i];
+    if (val < min) {
+      min = val;
+    }
+    if (val > max) {
+      max = val;
+    }
+  }
```

Comment:
forEach also lets you pass the index into the callback; any reason to prefer a `for` loop over it?

Reply:
Answered my own question, `for` loops are more efficient, but there's some debate over code clarity

Reply:
We found yesterday that using `forEach` was adding ~40ms to data loads from the cache (it would probably add more for volumes with more channels). Loads already (I think) feel a bit hitchy when trying to move quickly through a time series. I believe I wrote it this way from an earlier version that actually used `for` loops, so this is my mistake being corrected :)

Reply:
Yeah for looping over large numeric data arrays, there is a tangible performance gain to doing it this way.